magma_access_gateway_post_install/__init__.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+""" The Access Gateway (AGW) provides network services and policy enforcement. In an LTE network,
+  the AGW implements an evolved packet core (EPC), and a combination of an AAA and a PGW. It works
+  with existing, unmodified commercial radio hardware.
+  For detailed description visit https://docs.magmacore.org/docs/next/lte/architecture_overview.
+"""
+
+import logging
+import sys
+from argparse import ArgumentParser
+
+from systemd.journal import JournalHandler  # type: ignore[import]
+
+from .agw_post_install import AGWPostInstallChecks
+
+logger = logging.getLogger(__name__)
+handler = JournalHandler()
+handler.setFormatter(logging.Formatter("Magma AGW Post-Install: %(message)s"))
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+def main():
+    args = cli_arguments_parser(sys.argv[1:])
+
+    agw_post_install_checks = AGWPostInstallChecks(args.root_ca_path)
+
+    logger.info("Starting Magma AGW post-installation checks...")
+    agw_post_install_checks.prepare_system_for_post_install_checks()
+    agw_post_install_checks.check_whether_required_interfaces_are_configured()
+    agw_post_install_checks.check_eth0_internet_connectivity()
+    agw_post_install_checks.start_magma_service()
```

Comment:
Are these just post-install checks? Should we take actions like starting services?

Reply:
This part wasn't even needed. I just removed it in the wrong repo...
Fixed.